src/main/java/com/conveyal/datatools/manager/jobs/MergeFeedsJob.java

```diff
@@ -186,92 +133,118 @@ public MergeFeedsJob(Auth0UserProfile owner, Set<FeedVersion> feedVersions, Stri
         // permanently.
-        if (!mergedTempFile.delete()) {
-            // FIXME: send to bugsnag?
-            LOG.error(
+        try {
+            Files.delete(mergedTempFile.toPath());
+        } catch (IOException e) {
+            logAndReportToBugsnag(
+                e,
                 "Merged feed file {} not deleted. This may contribute to storage space shortages.",
-                mergedTempFile.getAbsolutePath());
+                mergedTempFile.getAbsolutePath()
+            );
         }
     }
 
     /**
      * Primary job logic handles collecting and sorting versions, creating a merged table for all versions, and writing
      * the resulting zip file to storage.
      */
-    @Override public void jobLogic() throws IOException, CheckedAWSException {
+    @Override
+    public void jobLogic() {
         // Create temp zip file to add merged feed content to.
-        mergedTempFile = File.createTempFile(filename, null);
-        mergedTempFile.deleteOnExit();
-        // Create the zipfile.
-        ZipOutputStream out = new ZipOutputStream(new FileOutputStream(mergedTempFile));
-        LOG.info("Created merge file: " + mergedTempFile.getAbsolutePath());
-        List<FeedToMerge> feedsToMerge = collectAndSortFeeds(feedVersions);
+        try {
+            mergedTempFile = File.createTempFile(filename, null);
+        } catch (IOException e) {
 
-        // Determine which tables to merge (only merge GTFS+ tables for MTC extension).
-        final List<Table> tablesToMerge =
-            Arrays.stream(Table.tablesInOrder)
-                .filter(Table::isSpecTable)
-                .collect(Collectors.toList());
-        if (DataManager.isExtensionEnabled("mtc")) {
-            // Merge GTFS+ tables only if MTC extension is enabled. We should do this for both
-            // regional and MTC merge strategies.
-            tablesToMerge.addAll(Arrays.asList(GtfsPlusTable.tables));
+            String message = "Error creating temp file for feed merge.";
+            logAndReportToBugsnag(e, message);
+            status.fail(message, e);
         }
-        int numberOfTables = tablesToMerge.size();
-        // Loop over GTFS tables and merge each feed one table at a time.
-        for (int i = 0; i < numberOfTables; i++) {
-            Table table = tablesToMerge.get(i);
-            if (mergeType.equals(REGIONAL) && table.name.equals(Table.FEED_INFO.name)) {
-                // It does not make sense to include the feed_info table when performing a
-                // regional feed merge because this file is intended to contain data specific to
-                // a single agency feed.
-                // TODO: Perhaps future work can generate a special feed_info file for the merged
-                //  file.
-                LOG.warn("Skipping feed_info table for regional merge.");
-                continue;
+        mergedTempFile.deleteOnExit();
```

Comment:
This can be moved as it is explictly handled in `jobFinished()` with Bugsnag error message.

Reply:
Thanks for pointing these out, I updated the code in https://github.com/ibi-group/datatools-server/pull/376/commits/225eb76e5e4b884cfdbe4d0e97ad2d3bceca7251.